manifests/0000_50_olm_10-operatorgroup.crd.yaml

```diff
@@ -91,7 +91,6 @@ spec:
                 type: string
               type: array
           required:
-          - namespaces
```

Comment:
This change is probably ok, but should be automatically updated next release.

Reply:
I ran `make release` given in the past when I changed something in template I was told to run `make release`...

Reply:
Right, so just get rid of the other changes. Running make release is an easy way to update this directory, but also updates the other directories which we don't want.